app/stores/NetworkStatusStore.js

```diff
@@ -25,9 +26,17 @@ export default class NetworkStatusStore extends Store {
   }
 
   @computed get syncPercentage(): number {
-    if (this.networkDifficulty > 0 && this._localDifficultyStartedWith != null) {
+    if (this.networkDifficulty > 0 && this._localDifficultyStartedWith !== null) {
       const relativeLocal = this.localDifficulty - this._localDifficultyStartedWith;
       const relativeNetwork = this.networkDifficulty - this._localDifficultyStartedWith;
+      // In case node is in sync after first local difficulty messages
+      // local and network difficulty will be the same (0)
+      console.log('networkDifficulty', this.networkDifficulty);
+      console.log('localDifficulty', this.localDifficulty);
+      console.log('relativeLocal', relativeLocal);
+      console.log('relativeNetwork', relativeNetwork);
+
+      if (relativeLocal === relativeNetwork) return 100;
       return relativeLocal / relativeNetwork * 100;
```

Comment:
Ah yeah, because network difficulty can never be less than local. Either add a comment about that or the check 😉